docs/pages/machine-id/guides/databases.mdx

```diff
@@ -230,6 +232,7 @@ Finally, run the following commands to start Machine ID:
 
 ```code
 $ sudo systemctl daemon-reload 
```

Comment:
Why not use the new partial you created here?

Reply:
This example includes two machine id services, machine-id and machine-id-proxy.